appshell/client_handler.cpp

```diff
@@ -202,7 +202,11 @@ bool ClientHandler::OnDragEnter(CefRefPtr<CefBrowser> browser,
 }
 
 void ClientHandler::OnLoadStart(CefRefPtr<CefBrowser> browser,
-                                CefRefPtr<CefFrame> frame) {
+                                CefRefPtr<CefFrame> frame
+                                #ifdef OS_LINUX
```

Comment:
is the function's signature like this for linux plaform alone?

Reply:
This is required as we have migrated to a newer version of CEF and the signature changed in the newer version.

Reply:
got you. this will have to be removed after we are on 2785 on all three platforms.